lib/remote.provider.js

```diff
@@ -1,7 +1,104 @@
 'use strict';
 
+const jstp = require('@metarhia/jstp');
+
 const { StorageProvider } = require('./provider');
+const { RemoteCursor } = require('./remote.cursor');
+
+const APP_NAME = 'GLOBAL_STORAGE';
+const INTERFACES = ['provider', 'cursor'];
+
+class RemoteProvider extends StorageProvider {
+  // Create RemoteProvider
+  // connection - jstp connection
```

Comment:
I don't think there is any need to add these comments describing the fields that you add in the constructor.